scipy/linalg/_sketches.py

```diff
@@ -45,6 +44,7 @@ def cwt_matrix(n_rows, n_columns, rng=None):
     .. math:: \|SA\| = (1 \pm \epsilon)\|A\|
     Where the error epsilon is related to the size of S.
     """
+    from scipy.sparse import csc_matrix
```

Comment:
Would be nice to have an explanatory comment, potentially with a link to the gh issue.

So that it's clearer to future contributors that it's a deliberate exception to the rule that by default all imports are eager and are at the top level.